app/src/main/res/layout/activity_help_feedback.xml

```diff
@@ -4,4 +4,18 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
+    <TextView
+        android:id="@+id/textView5"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="20dp"
+        android:layout_marginTop="50dp"
+        android:text="For all your querries and feedback click the link below:" />
```

Comment:
This can be placed inside `strings.xml` files and it should be `queries` not `querries`